dynawo/sources/Models/Modelica/Dynawo/Examples/DynaFlow/IEEE14/TestCases/IEEE14NoEvent.mo

```diff
@@ -0,0 +1,195 @@
+within Dynawo.Examples.DynaFlow.IEEE14.TestCases;
```

Comment:
This test case doesn't simulate on my computer, how about you ? Maybe we can remove it and just keep the DisconnectLine one ? What do you think ?